packages/react/src/SubdomainNavBar/SubdomainNavBar.tsx

```diff
@@ -369,6 +450,15 @@ const _SearchInternal = (
                 </ul>
               </div>
             )}
+            <div
+              aria-live="polite"
+              aria-atomic="true"
+              data-testid="search-live-region"
+              className={styles['SubdomainNavBar-search-result-item-suggestion']}
+            >
+              {`${searchResults?.length} suggestions.`}
```

Comment:
I like this feature, and wonder if we can make this work for sighted users also..

@danielguillan thoughts on putting the results count in the results heading?

Before:
![Screenshot 2023-04-11 at 12 08 20](https://user-images.githubusercontent.com/13340707/231142963-d42dc946-458a-4842-bb13-f1bc93da6073.png)

After:
![Screenshot 2023-04-11 at 12 11 19](https://user-images.githubusercontent.com/13340707/231143092-68f25721-11e5-49dd-94bd-e82c5db0fe05.png)

@TylerJDev - if we did this ☝️ - would that make this visually hidden code redundant?

Reply:
Yup! We could definitely remove the current visually hidden live region and utilize this instead if we went this route. This might even be a more accessible way of going about it too!

Reply:
cc. @josepmartins @danielguillan .. would you both be okay to move forward with this?